mathics/builtin/arithfns/basic.py

```diff
@@ -289,22 +289,15 @@ def format_plus(self, items, evaluation):
         "Plus[items__]"
 
         def negate(item):
-            if item.has_form("Times", 1, None):
-                if isinstance(item.leaves[0], Number):
-                    neg = -item.leaves[0]
-                    if neg.sameQ(Integer1):
-                        if len(item.leaves) == 1:
-                            return neg
-                        else:
-                            return Expression("Times", *item.leaves[1:])
-                    else:
-                        return Expression("Times", neg, *item.leaves[1:])
+            neg: Integer = -item.leaves[0]
+
+            if neg.sameQ(Integer1):
+                if len(item.leaves) == 1:
+                    return neg
                 else:
-                    return Expression("Times", -1, *item.leaves)
-            elif isinstance(item, Number):
```

Comment:
It is never a `Number`.

Reply:
That is never a `Number` because we only want to negate `x + -y` and `-y` is the same as `Times[-1, y]`.